webpack.config.js

```diff
@@ -0,0 +1,55 @@
+const path = require("path");
+const version = process.env.VERSION;
+let watch = true;
+
+if( version === "Prod" || version === "Template"){ 
+    watch = false;
+}
+
+module.exports = {
+	mode: "production",
+    entry: {
+        launch: path.resolve(__dirname, "src/launch.js"),
+        index: path.resolve(__dirname, "src/index.js"),
+        register: path.resolve(__dirname, "src/register.js"),
+        log: path.resolve(__dirname, "src/logs.js")
+      },
+      output: {
+        filename: "[name].bundle.js",
+        path: path.resolve(__dirname, "public/js"),
+        publicPath: "/"
+      },
+	watch: watch,
+    resolve: { extensions: ["*", ".js", ".jsx"] },
```

Comment:
most of this file is the same as our old `webpack.config.common.js`, with the key changes being that it now conditionally uses the `watch` parameter to rebuild the files in a dev environment.